internal/pagination/pagination.go

```diff
@@ -0,0 +1,53 @@
+package pagination
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/rs/zerolog"
+)
+
+type ctxKeyType int
+
+const (
+	defaultValueLimit             = 100
+	defaultValueOffset            = 0
+	offsetCtxKey       ctxKeyType = iota
+	limitCtxKey
+)
```

Comment:
Can you please create a dedicated `const` block for default values (the first two lines)? I am not sure if it is a good practice to mix iotas with other values.